lib/vet/tarball.js

```diff
@@ -9,16 +9,24 @@ module.exports = function(module, dir, url, json) {
   var moduleDir = path.join(dir, 'node_modules', module);
   return new Promise(function(resolve, reject) {
     var children = [];
-    var stream = got.stream(url).pipe(zlib.Unzip()).pipe(tar.Parse());
+    var stream = got.stream(url).pipe(zlib.Unzip()).pipe(new tar.Parse());
```

Comment:
this is an api change from newer module?

Reply:
Yes.